Changelog.md

```diff
@@ -1,6 +1,45 @@
 # 0.3.0.0
 
-Numerous changes.
+ - LSP mode is now the default, and the `--lsp` flag has no effect
+   - The `--json` flag can be used for JSON transport
+ - HIE now warns you if there is mismatch between the HIE GHC verison and the project GHC version
+ - Add Liquid Haskell support
+ - Add support for hierarchical document symbols
+ - Add many new types of code actions
+   - Typed holes
+   - HaRe refactoring
+   - Misspelled variables
+   - Missing top-level signatures
+   - Prefix unused terms with '_'
```

Comment:
Using different apostrophe character seems to break markdown formatting. Please use the first one in both places on line 13